### ISSUE TITLE
Some questions?

### ISSUE THREAD

**Issue by maojinyuan** [glotzerlab/hoomd-blue#900]
Thanks for a gorgeous project of Hoomd-Blue firstly. I get some trouble when installing...

1. Is the hoomd-blue a pre-build software? cause when I try to compiling from source instead of installing by conda, I can import hoomd in python3 when I enter the sub-folders. Is that possible?

2. After 1, I have compiled the source, and no error happened, but I can't import hoomd in python3 interpreter out of the hoomd folders(error for " No module named 'hoomd'"), I supposed maybe my enviroment is not correct. for example:
`$ source /usr/local/bin/activate`
`$ export CMAKE_PREFIX_PATH=/usr/local`
but for the first line of code, error for "No such file or directory"
so what should I do to sovle the enviroment problem?

3. If we can import others plugins from conda platform?

I'm looking forward your suggestions.Thx😘

**Comment by b-butler** [comment]
@maojinyuan when it compiled did you run `make install`?

**Comment by maojinyuan** [comment]
Absolutely yes, I followed as the manual said.

**Comment by b-butler** [comment]
@maojinyuan sorry for taking so long to respond back, but without more information about the exact setup you have and the commands you ran to install HOOMD-blue. It will be hard to help.

As regards your first comment above
> `$ source /usr/local/bin/activate`
For this to work you had to have set up a virtual environment at `/usr/local/bin` which is unlikely given that that is a systems folder.

**Comment by maojinyuan** [comment]
@b-butler Hi, Thanks for your suggestions. 

Now, I've solved the problem, and I summary the conclusions of this process:
Many of researcher fork the project only in the older version of Hoomd-Blue, so I ignored the incompatible version between Hoomd-Blue and cuda-toolkit.
I do believe that the new version of cuda-toolkit(version 11.2)  did not support the old version(e.g. 2.8.2) of Hoomd-Blue. I just test the Hoomd-Blue(version 2.8.2) successfully in the version of cuda-toolkit 10.1&10.2.

I write down here for the later researcher to refer, and it is inspiring if you'd like to solve the incompatible problem.

Anyway, Thanks 👍:)
